plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -88,11 +88,41 @@ Transliterate string from cyrillic russian to latin
 };
 
 /*
+Given a list of filters, apply every one in turn to source, and return the first result of the first filter with non-empty result.
+*/
+FileSystemAdaptor.prototype.findFirstFilter = function(filters, source) {
```

Comment:
Our coding style is a little unusual, I'm afraid, and still a little inconsistent. Anyhow, we don't use spaces within parameter lists in function calls or definitions.